modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-azure.adoc

```diff
@@ -0,0 +1,183 @@
+
+* When the deployment is complete click on Go to Resource
+
+image::{dir}/azure-app-service-deployment-complete-5.PNG[Create Azure web app Complete,align="center" width=75%]
+
+=== Configure Jupyter Lab extensions Azure Web App
+* Select Configuration on the right panel
+
+image::{dir}/azure-app-service-resource-6.PNG[Create Azure web app Complete,align="center" width=75%]
+
+* Add the following Application Settings
+
++
+[cols="1,1"]
+|====
+| *Application Setting* | *Value*
+| *accept_license* 
+| Y
+| *WEBSITES_PORT* 
+| 8888
+| *JUPYTER_TOKEN*
+| Your desired Jupyter Lab token, if not added, Jupyter Lab will randomly generate one, requiring perusing the logs, if left blank Jupyter Lab the environment would not be locked
+|====
+
+* Click on save, your app will be restarted
+
+image::{dir}/azure-app-service-resource-config-7.PNG[Config Azure web app,align="center" width=75%]
+
+* Return to the Overview tab on the right panel
+
+=== Load Vantagecloud Lake demos to Jupyter Lab extensions Azure Web App
+* Click on Default domain
+
+image::{dir}/azure-app-service-resource-8.PNG[Config Azure web app,align="center" width=75%]
+
+* On the Jupyter Lab start dialogue enter the defined Jupyter token and click Log in
+
+image::{dir}/azure-jupyter-console-auth-9.PNG[Config Azure web app,align="center" width=75%]
+
+* On the Jupyter Lab console click on the git icon
+
+image::{dir}/azure-jupyter-console-10.PNG[Config Azure web app,align="center" width=75%]
+
+* Copy the following URI in the corresponding field
+** `https://github.com/Teradata/lake-demos.git`
+* Click Clone
+
+image::{dir}/azure-jupyter-console-clone-11.PNG[Config Azure web app,align="center" width=75%]
+
+* On the Jupyter Lab console click in the Lake Demos folder
+
+image::{dir}/azure-jupyter-console-click-lake-demos-12.PNG[Config Azure web app,align="center" width=75%]
+
+image::{dir}/azure-jupyter-console-lakedemos-13.PNG[Config Azure web app,align="center" width=75%]
+
+=== Find the IP of the Jupyter Lab extensions Azure Web App
+* On JupyterLab open a notebook with Teradata Python kernel and run the following command for finding your notebook instance IP address.
+
++
+[source, python, id="lakedemos_azure_first_config", role="emits-gtm-events, content-editable"]
+----
+import requests
+def get_public_ip():
+    try:
+        response = requests.get('https://api.ipify.org')
+        return response.text
+    except requests.RequestException as e:
+        return "Error: " + str(e)
+my_public_ip = get_public_ip()
+print("My Public IP is:", my_public_ip)
+----
+
+** We will whitelist this IP in your VantageCloud Lake environment in order to allow the connection
+** This is for purposes of this guide and the notebooks demos. For production environments, a more robust networking setting might be needed
+** Azure App Service offers, as well, a list of all possible IP addresses that the service might expose. This is under the overview tab
```

Comment:
I suggest we adjust the verbiage a bit. Also, should this not be a callout instead?

```suggestion
** The next step is whitelist this IP in your VantageCloud Lake environment to allow the connection
** This is for purposes of this guide and the notebook demos. For production environments, a more robust networking setting might be needed
** Azure App Service offers, as well, a list of all possible IP addresses that the service might expose. This is under the overview tab
```